frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -268,6 +268,26 @@ pub type CompactAccuracyOf<T> = <CompactOf<T> as CompactSolution>::Accuracy;
 /// The accuracy of the election, when computed on-chain. Equal to [`Config::OnChainAccuracy`].
 pub type OnChainAccuracyOf<T> = <T as Config>::OnChainAccuracy;
 
+/// A voter's fundamental data: their ID, their stake, and the list of candidates for whom they voted.
+pub type Voter<T> = (
```

Comment:
these types are internal to the miner and unsigned stuff, I'd move them to `unsigned.rs` to keep things clean.

Reply:
0c76b221d